app.rb

```diff
@@ -1,20 +1,64 @@
 require_relative "data"
-apartments = data[:apartments]
-tenants = data[:tenants]
-
+ruby_apartments = data[:apartments]
+ruby_tenants = data[:tenants]
+#
 # First, Open the data.rb an inspect the data. Identify and write, in comments, the following:
```

Comment:
the idea is to use the existing `apartments` and `tenants` arrays to reconstruct the data using ruby class objects